x/ibc/02-client/exported/exported.go

```diff
@@ -25,6 +25,7 @@ type ClientState interface {
 	// State verification functions
 
 	VerifyClientConsensusState(
+		store sdk.KVStore,
```

Comment:
replaced `sdk.Context` with `KVStore` since the client won't be able to access the store thought the context unless we passed the IBC StoreKey as an argument. 

This approach is more secure because we can just pass the prefix store for the specific client.